docs/public/_redirects

```diff
@@ -378,18 +378,28 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /:lang/joy-ui/core-features/global-variant/ /:langjoy-ui/main-features/global-variants/ 301
 /joy-ui/core-features/automatic-adjustment/ /joy-ui/main-features/automatic-adjustment/ 301
 /:lang/joy-ui/core-features/automatic-adjustment/ /:lang/joy-ui/main-features/automatic-adjustment/ 301
-/joy-ui/core-features/perfect-dark-mode/ /joy-ui/main-features/perfect-dark-mode/ 301
-/:lang/joy-ui/core-features/perfect-dark-mode/ /:lang/joy-ui/main-features/perfect-dark-mode/ 301
+/joy-ui/core-features/perfect-dark-mode/ /joy-ui/main-features/dark-mode-optimization/ 301
+/:lang/joy-ui/core-features/perfect-dark-mode/ /:lang/joy-ui/main-features/dark-mode-optimization/ 301
 /system/basics/ /system/getting-started/overview/ 301
+/:lang/system/basics/ /:lang/system/getting-started/overview/ 301
 /system/the-sx-prop/ /system/getting-started/the-sx-prop/ 301
+/:lang/system/the-sx-prop/ /:lang/system/getting-started/the-sx-prop/ 301
 /system/advanced/ /system/getting-started/custom-components/ 301
-/:lang/base/react-trap-focus/ /:lang/base/react-focus-trap/ 301
-/:lang/base/api/trap-focus/ /:lang/base/api/focus-trap/ 301
+/:lang/system/advanced/ /:lang/system/getting-started/custom-components/ 301
 /base/react-trap-focus/ /base/react-focus-trap/ 301
+/:lang/base/react-trap-focus/ /:lang/base/react-focus-trap/ 301
 /base/api/trap-focus/ /base/api/focus-trap/ 301
+/:lang/base/api/trap-focus/ /:lang/base/api/focus-trap/ 301
 /material-ui/experimental-api/css-variables/ /material-ui/experimental-api/css-theme-variables/overview/ 301
+/:lang/material-ui/experimental-api/css-variables/ /:lang/material-ui/experimental-api/css-theme-variables/overview/ 301
 /joy-ui/main-features/perfect-dark-mode/ /joy-ui/main-features/dark-mode-optimization/ 301
+/:lang/joy-ui/main-features/perfect-dark-mode/ /:lang/joy-ui/main-features/dark-mode-optimization/ 301
 /joy-ui/guides/applying-dark-mode/ /joy-ui/customization/dark-mode/ 301
+/:lang/joy-ui/guides/applying-dark-mode/ /:lang/joy-ui/customization/dark-mode/ 301
```

Comment:
We still generate the translation pages for the docs, redirecting the en-US locale isn't enough.

Once we stop this generation of pt, zh pages, we will be able to not have `/:lang/` for the redirection of pages first introduced after the generation stopped. So, it's not for tomorrow.